inst/app/ui.R

```diff
@@ -158,50 +161,50 @@ ui <- dashboardPage(
                   `actions-box` = TRUE,
                   `deselect-all-text` = "Unselect All",
                   `select-all-text` = "Select All",
-                  `none-selected-text` = "Select vehicle class(es)...",
+                  `none-selected-text` = "Select casualty role(s)...",
                   `selected-text-format` = "count",
-                  `count-selected-text` = "{0} vehicle classes choosed (on a total of {1})"
+                  `count-selected-text` = "{0} casualty roles choosed (on a total of {1})"
                 ),
                 choicesOpt = NULL,
                 width = NULL,
                 inline = FALSE
               ),
 
-              p("NOTE: Multiple selections mean the accident includes ANY of the selected classes (instead of includes ALL of the selected classes)."),
-
               pickerInput(
-                inputId = "severity_filter", label = "Accident Severity",
-                choices = unique(hk_accidents$Severity),
-                selected = unique(hk_accidents$Severity),
+                inputId = "vehicle_class_filter", label = "Vehicle classes involved in the collision",
+                choices = unique(hk_vehicles$Vehicle_Class),
+                selected = unique(hk_vehicles$Vehicle_Class),
                 multiple = TRUE,
                 options = list(
                   `actions-box` = TRUE,
                   `deselect-all-text` = "Unselect All",
                   `select-all-text` = "Select All",
-                  `none-selected-text` = "Select Severity type(s)...",
+                  `none-selected-text` = "Select vehicle class(es)...",
                   `selected-text-format` = "count",
-                  `count-selected-text` = "{0} severity types choosed (on a total of {1})"
+                  `count-selected-text` = "{0} vehicle classes choosed (on a total of {1})"
                 ),
                 choicesOpt = NULL,
                 width = NULL,
                 inline = FALSE
               ),
 
-              # Multiple UI choices available for this filter
-              # TODO: select between `checkboxGroupInput` or `checkboxGroupButtons` as UI of this filter
-              checkboxGroupButtons(
-                inputId = "pedestrian_involved_filter", label = "Pedestrian Involved in the accident?",
-                choices = list("Yes" = TRUE, "No" = FALSE),
-                selected = list(TRUE, FALSE),
-                justified = TRUE, status = "primary"
+              p("NOTE: Multiple selections means filtering accidents including ANY selected classes
+                (instead of ALL selected classes)."),
+
+              sliderInput(
+                inputId = "n_causality_filter", label = "No. of casualties",
+                min = min(hk_accidents$No__of_Casualties_Injured),
+                max = max(hk_accidents$No__of_Casualties_Injured),
+                value = range(hk_accidents$No__of_Casualties_Injured),
+                step = 1
```

Comment:
Should the range of 1 to 77 update according to the filters chosen by the user? If yes, it didn't work when I tried.